spicy/toolchain/bin/spicy-config.cc

```diff
@@ -222,6 +223,15 @@ int main(int argc, char** argv) {
 #endif
         }
 
+        if ( opt == "--zeek-version" ) {
+#ifdef HAVE_ZEEK
+            result.emplace_back(ZEEK_VERSION_NUMBER);
+#else
+            result.emplace_back("0");
```

Comment:
Can we make this an error instead of making up a value which looks valid?

Reply:
It's convenient this way because it allows to check against an expected Zeek version number even if Zeek isn't available at all (with the right result).